docs/topics/scripting/custom-script-deps-tutorial.md

```diff
@@ -0,0 +1,332 @@
+[//]: # (title: Get started with Kotlin custom scripting – tutorial)
+
+> Kotlin scripting in [Experimental](components-stability.md). It may be dropped or changed at any time.
+> Use it only for evaluation purposes. We appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+>
+{type="warning"}
+
+_Kotlin scripting_ is the technology that enables executing Kotlin code as scripts, without prior compilation or
+packaging into executables.
```

Comment:
If the Scripting is located in the Platforms section, maybe it's better to write explicitly that it's "platform independent" or smth similar?

Reply:
If I get things right, the host we're creating here works only on the JVM.
@ligee please correct me if I'm wrong.

Reply:
Yes, although we have some basic support for JS scripting, it cannot be considered even experimental, so do not want to mention it yet in any form.